app/front/scripts/services/navigation.js

```diff
@@ -28,7 +28,7 @@
           filterList.push(key+'|'+value);
         });
 
-        var embed = (state.isEmbeded)? '/embed': '';
+        var embed = (state.isEmbedded)? '/embed': '';
```

Comment:
@pwalsh I have seen new pull request from Adam  about base url and reletive urls. so, your way is better. I will take Adam's changes and will show/hide header and footer on backend.